js/src/controls/SearchControl.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+const L = require('../leaflet.js');
+const control = require('./Control.js');
+
+export class LeafletSearchControlModel extends control.LeafletControlModel {
+  defaults(){
+    return{
+      _view_name: 'LeafletSearchControlView',
+      _model_name: 'LeafletSearchControlModel',
+      url: null,
+      jsonpParam: 'json_callback',
```

Comment:
Same here, they should reflect the Python attribute names here.

Reply:
Done.